### PR TITLE
Add runtimes : { win : {} } to project.json files

### PR DESCRIPTION
**Pull request by mjrousos**
This is necessary to meet VS2015 Update 1 tools requirements

### CODE REVIEW

File: src/ApiPort/project.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ﻿{
   "runtimes": {
-    "win-anycpu": { }
+    "win-anycpu": { },
+    "win": { }
```
Comment (twsouthwick) [inline src/ApiPort/project.json]: Do we need both of these?

**Comment by twsouthwick** [comment]
LGTM - should we wait until RTM of Update 1? In either case, please update the README.md to clarify that Update 1 is now required.

File: src/ApiPort/project.json (hunk 1 above)
Comment (mjrousos) [inline src/ApiPort/project.json]: I left the old one because Microsoft/msbuild#344 mentioned that the win- style is needed to build with RTM.

**Comment by mjrousos** [comment]
Ah, does this break building with RTM? If so, we should wait for Update 1 to release (RTM is such an overloaded term here :). I will test it out; I thought by not removing the old win- runtimes, it would still build without update 1 but I need to dig up a non-upgraded machine to try it.

File: src/ApiPort/project.json (hunk 1 above)
Comment (mjrousos) [inline src/ApiPort/project.json]: I checked and it looks like win-anycpu is still needed to build on 2015 RTM

**Comment by mjrousos** [comment]
I tried it out and these projects seem to build fine on both RTM and Update 1 once the project.json files are updated.

**Pull request merged**